db/migrate/20230214154931_create_places.rb

```diff
@@ -0,0 +1,14 @@
+class CreatePlaces < ActiveRecord::Migration[7.0]
+  def change
+    create_table :places, id: :uuid do |t|
+      t.string :name
+      t.string :tags
+      t.text :deatails
+      t.string :image
+      t.string :address
+      t.string :location
```

Comment:
It is not correct. You should create a relationship between the address and location model not set them as string attributes.